recipe/recipe.yaml

```diff
@@ -328,6 +330,7 @@ outputs:
         - jupyterlab_execute_time >=3.2.0
         - jupyterlab_iframe >=0.5.0
         - jupyterlab_widgets >=3.0.16
+        - libpq==17.6
```

Comment:
This is *still* in the wrong place.  This needs to be in the main constraint.  Please look at the build logs and you can see the libpq version mismatch.